bettercli/cl/input.py

```
@@ -0,0 +1,236 @@
+from functools import wraps
+import sys
+import keyboard
+import time
+import typing as t
+from . import colours as c
+
+T = t.TypeVar("T", bound=dict)
+
+
+class Dict(dict, t.Generic[T]):
+    def __init__(self, dict:'dict', default:'dict'={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)